composer.json

```diff
@@ -13,6 +13,7 @@
         "fruitcake/laravel-cors": "^1.0",
         "guzzlehttp/guzzle": "^6.3",
         "laravel/framework": "^7.0",
+        "laravel/helpers": "^1.2",
```

Comment:
zasta koristis ovaj paket ?

Reply:
to je trebalo da mi asistira sa intelephense erorima.